dev-python/pipenv/pipenv-2020.4.1_beta_p1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python Development Workflow for Humans"
+HOMEPAGE="https://github.com/pypa/pipenv https://pypi.org/project/pipenv/"
+SRC_URI="https://github.com/pypa/pipenv/archive/v2020.4.1b1.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="test"
+
+PATCHES=(
+	"${FILESDIR}/distable-networked-tests.patch"
+	)
+
+RDEPEND="
+	dev-python/certifi[${PYTHON_USEDEP}]
+	dev-python/virtualenv[${PYTHON_USEDEP}]
+	>=dev-python/virtualenv-clone-0.2.5[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	>=dev-python/pip-18.0[${PYTHON_USEDEP}]
+	>=dev-python/setuptools-36.2.1[${PYTHON_USEDEP}]
+	"
+
+DEPEND="
+	test? (
+		${RDEPEND}
+		<dev-python/pytest-5.0[${PYTHON_USEDEP}]
+		dev-python/pytest-timeout[${PYTHON_USEDEP}]
+		dev-python/pytest-xdist[${PYTHON_USEDEP}]
+		dev-python/flaky[${PYTHON_USEDEP}]
+		dev-python/flask[${PYTHON_USEDEP}]
+		dev-python/mock[${PYTHON_USEDEP}]
+		dev-python/pytz[${PYTHON_USEDEP}]
+		)"
+
+S=${WORKDIR}/pipenv-2020.4.1b1
+
+python_test() {
+	pytest  -m "not cli"  -vvv -s -x tests/unit || die
```

Comment:
Eclass `distutils-r1` de-facto recommends `pytest -vv` and the current `pipenv-2018.11.26.ebuild` has that. So I'm not sure we'll want to add `-x|--exitfirst`, another `-v` and `-s|--capture=no` here.
The double space looks mistaken.
Why excluse `cli` tests?

Reply:
All the cli test require network. Unfortunately.

Reply:
I changed the cli options to pytest according to the recommendations.